livre/eq/render/RayCastRenderer.cpp

```diff
@@ -430,9 +429,13 @@ struct RayCastRenderer::Impl
 
     void onFrameRender( const NodeIds& bricks )
     {
+        createAndFillVertexBuffer( bricks );
         size_t index = 0;
         for( const NodeId& brick: bricks )
             renderBrick( brick, index++ );
+
+        glDeleteBuffers( 1, &_posVBO );
+        glFlush();
```

Comment:
- Yes, it is a one-liner. glDeleteBuffer(...)
- I don't think it is a bad design.. OpenGL functions, malloc, fopen etc all uses the same idea. You have to delete them yourself. But those are not C++ idioms. In a perfect world VBO is an C++ object that handles this situation but we don't use wrapping here.